opus/application/static_media/js/utils.js

```diff
@@ -109,7 +109,18 @@ var o_utils = {
     // or inner objects. In our case, we only have arrays as values.
     deepCloneObj: function(obj) {
         return JSON.parse(JSON.stringify(obj));
-    }
+    },
+
+    getSurfacegeoTargetSlug: function(target) {
```

Comment:
Maybe put in a comment here referencing the file in the import pipeline. Something like "This function is patterned after slug_name_for_sfc_target in import_util.py".